packages/odyssey-react/stylelint.config.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
         ignoreProperties: [ 'composes' ]
       }
     ],
+    'no-invalid-double-slash-comments': true,
```

Comment:
see my question above about the lint change here around comments

Reply:
Bah, this is unintentional. This rule doesn't accomplish what I'm trying to. Will remove.